lib/druid_tools/druid.rb

```diff
@@ -122,6 +126,7 @@ def mkdir_with_final_link(source, extra = nil)
       FileUtils.mkdir_p(real_path)
       FileUtils.ln_s(source, new_path, force: true)
     end
+    deprecation_deprecate :mkdir_with_final_link
```

Comment:
Should these include some text about what to use instead?

Reply:
They aren't being replaced.